examples/jsm/controls/OrbitControls.js

```diff
@@ -114,6 +114,19 @@ class OrbitControls extends EventDispatcher {
 
 		};
 
+		this.getDistance = function () {
```

Comment:
```js
this.getDistance = function () {

    return this.object.position.distanceTo( this.target );

}
```

Reply:
This is how I would do it if @Mugen87 agrees.

Reply:
👍